instrument/D17_Parameters.xml

```diff
@@ -22,14 +22,15 @@
 		</parameter>
 		<!-- Default monitor spectrum number for monitor normalisation. -->
 		<parameter name="default-incident-monitor-spectrum" type="int" visible="false">
-			<value val="257" />
+			<value val="256" />
```

Comment:
This was incorrectly filled from the beginning, so that's a correction of an error. It fixes the normalisation procedure, so perhaps a release note could be added.